staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret.go

```diff
@@ -44,10 +44,11 @@ import (
 // This is the entry point of create_secret.go which will be called by create.go
 func NewCmdCreateSecret(f cmdutil.Factory, ioStreams genericiooptions.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "secret",
-		Short: i18n.T("Create a secret using specified subcommand"),
-		Long:  i18n.T("Create a secret using specified subcommand."),
-		Run:   cmdutil.DefaultSubCommandRun(ioStreams.ErrOut),
+		Use:                   "secret (docker-registry | generic | tls)",
+		DisableFlagsInUseLine: true,
+		Short:                 i18n.T("Create a secret using specified subcommand."),
```

Comment:
Nit:

```suggestion
		Short:                 i18n.T("Create a secret using a specified subcommand"),
```

This follows the existing pattern used by service...
```
  service               Create a service using a specified subcommand
```

Reply:
Since we are talking about this, I found that `create service --help` command's description may be too simple? I've already enriched `create secret --help` from [this](https://github.com/kubernetes/kubernetes/pull/117930/files#diff-de18eda42b829e371e14d98fec038cee9ff68b6c0d85206614fe8f972d18d95aL49) to [this](https://github.com/kubernetes/kubernetes/pull/117930/files/fd291fa6600f54cdad79a39fbb30904080e3cc84#diff-de18eda42b829e371e14d98fec038cee9ff68b6c0d85206614fe8f972d18d95aR62) while `create service --help` still a simple sentence, what you think？

Reply:
Moreover, `create service` `plugin` `rollout` `set` `top` those commands' *Usage* line are all like `create secret`'s behavior that I want to improve here. They all show `kubectl xxx [flags] [options]`. In my opinion, they should show like `kubectl xxx (subcommand1 | subcommand2 | subcommand3) [options]` in an enum style, or at least `kubectl xxx <command> [options]`. Speaking of this, the `rollout` and `set` commands even behaves differently than other commands: 
```shell 
kubectl rollout --help
...
Usage:
  kubectl rollout SUBCOMMAND [options]
...
```
they just show "SUBCOMMAND". Should we optimize these contents? In which way? Need your advice 😆

Reply:
I'm sure there are plenty of inconsistencies.

I would say let's focus on this change now, and then if there is more work needed to make help messages consistent, then do that as a follow-up PR.

If the changes more significant than just making command help consistent with each other, then I would say create an issue for discussion, so you can make sure that the proposed changes are agreed upon before doing a bunch of work.  For example, the enum style would probably be something that the sig should decide on, so either an issue and/or a topic in the bi-weekly sig meeting would be appropriate for that,

Reply:
I see your point and agree with you.

> I would say let's focus on this change now, and then if there is more work needed to make help messages consistent, then do that as a follow-up PR.

I'll finish this PR first and then discuss the rest in an issue.